lua/formations.lua

```diff
@@ -449,7 +449,7 @@ local GrowthChevronBlock = {
 
 local LightAttackNaval = categories.LIGHTBOAT
 local FrigateNaval = categories.FRIGATE
-local SubNaval = categories.T1SUBMARINE + categories.T2SUBMARINE + (categories.NUKESUB * categories.ANTINAVY - categories.NUKE)
+local SubNaval = categories.T1SUBMARINE + categories.T2SUBMARINE + (categories.SERAPHIM * categories.TECH3 * categories.SUBMERSIBLE)
```

Comment:
```suggestion
local SubNaval = categories.T1SUBMARINE + categories.T2SUBMARINE + (categories.TECH3 * categories.SUBMERSIBLE * categories.NAVAL - categories.NUKE)
```
This would be more general-purpose for mod compatibility. `NAVAL` category excludes HARMS. 
This would still catch *any* submersible tech 3 units, e.g. a submerging battleship like Tempest but at tech 3, which is why I could see a `T3SUBMARINE` category being useful.